src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/EnvironmentSetupHelper.cs

```diff
@@ -395,6 +395,7 @@ public virtual Collection<PSObject> RunPowerShellTest(params string[] scripts)
         private void SetupPowerShellModules(System.Management.Automation.PowerShell powershell)
         {
             powershell.AddScript("$error.clear()");
+            powershell.AddScript("Set-ExecutionPolicy Bypass -Scope Process");
```

Comment:
The test cases only run if I add that line. I usually don't submit the change but since I always have to do it and it doesn't harm I submitted it this time.